org.eclipse.winery.frontends/app/topologymodeler/src/app/edmmTransformationCheck/edmmTransformationCheck.component.html

```diff
@@ -29,7 +29,7 @@
                     </div>
                 </li>
             </ul>
-<!--            <button (click)="doTransformationCheck()">Do Check</button>-->
+            <!--            <button (click)="doTransformationCheck()">Do Check</button>-->
```

Comment:
why is this button disabled?

Reply:
This change is a result of auto formatting the whole file. The button was already disabled, no idea why

Reply:
@miwurster, do you know somthing about it?

Reply:
This can be ignored for now since it has been adapted and fixed on my fork ;-)